README.md

```diff
@@ -137,6 +137,18 @@ some important features.
 
 Any and all help is greatly appreciated!
 
+**NOTE:** `latexmk` does not support file paths containing special characters such as `~`. To partially circumvent this, add `useRelativePaths: true` to your config file like so
+```cson
+# config.cson
+"*":
+  "atom-latex":
```

Comment:
This should be `latex`

Reply:
I had a rogue `atom-latex` entry in my config, so I assumed it was like that. Fixed.